tests/test.cpp

```diff
@@ -568,6 +568,33 @@ void FuzzTest2() {
 
   RndDef definitions[num_definitions];
 
+  // Since we're generating schema and corresponding data in tandem,
+  // this convenience functor adds strings to both at once.
+  struct AddToSchemaAndInstances_t
```

Comment:
I think it be even shorter if instance variables be turned into arguments. Or at least more readable.